infra/monitoring/README.md

```diff
@@ -0,0 +1,7 @@
+# Monitoring Infra
+
+To setup an instance of the monitoring infrastructure:
+- create a GCP project and set it as L12N_GCP_PROJECT_ID
+- choose a GCP region and set it as L12N_GCP_REGION
+- before deploying, you must run `l12n monitoring.login` and authenticate
```

Comment:
⁉️ there is a missing task, I suppose. I'm getting
`hostcaller@b5672eb22b3e:path_to_lambdatization$ l12n monitoring.login

No idea what 'monitoring.login' is!`

Reply:
plugin wasn't activated. Once I set L12N_PLUGINS=monitoring the login process start but it fails to save the defaut credentials.
`ERROR: (gcloud.auth.application-default.login) Error saving Application Default Credentials: Could not create directory [/etc/persistent-configs/gcloud]: Permission denied.`

Reply:
This should be fixed by https://github.com/cloudfuse-io/lambdatization/commit/721d8a88e3e76d8fbaf4024310303f7c5db1dc35

Reply:
can you confirm?

Reply:
I still have the issue. Testing on 5e8c2115 I got the same error
`ERROR: (gcloud.auth.application-default.login) Error saving Application Default Credentials: Could not create directory [/etc/persistent-configs/gcloud]: Permission denied.`

Reply:
working now.